src/scss/_variables.scss

```diff
@@ -1,6 +1,6 @@
 @use 'sass:math';
 
-$fa-font-path: '../fonts/font-awesome';
+$fa-font-path: '../fonts/font-awesome-pro';
```

Comment:
This is probably okay for now, but what if the pro icons aren't available (because the person doesn't have the appropriate key). Should this variable be injected by the build script somehow?

Reply:
You are right, I'll try to fix this issue in another PR.